ld-relay.go

```diff
@@ -101,7 +110,39 @@ func main() {
 		}(envName, *envConfig)
 	}
 
-	wg.Wait()
+	http.Handle("/status", http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
```

Comment:
It is-- this isn't where that's handled though. See where the configuration port is usdd